### DISCUSSION TITLE
SRT for 1.2gbps streams

### DISCUSSION THREAD

**Discussion by ravtr** [Haivision/srt#2493] [General]
We are looking for a protocol to use to transmit a transport stream of around 1.2gbps over a lossy network.

From some blog post, its understood that SRT can support upto 1gbps and can be extended if needed.

Is there any benchmarking done for SRT wrt bitrate, latency and CPU wrt high bandwidth streams?

**Comment by maxsharabayko** [comment]
1 Gbps is just the default [SRTO_MAXBW](https://github.com/Haivision/srt/blob/master/docs/API/API-socket-options.md#SRTO_MAXBW) value in the live streaming configuration (TSBPD enabled) suitable for the most of use cases.
You can increase it however you want.

**Reply by code101-marko** [reply]
Hi @maxsharabayko does this needs to be modified to support >1Gbps when using srt-xtransmit for testing? Or will the SDK allow the maximum possible? Thanks!

**Reply by maxsharabayko** [reply]
You can just use the 'maxbw=bytes' URI query. See https://github.com/maxsharabayko/srt-xtransmit/blob/master/docs/uri-syntax.md

**Reply by code101-marko** [reply]
Hey @maxsharabayko , this did help. Regarding mss (SRTO_MSS), can this be increased over 1500 if ec2 is configured to use jumbo frames?

**Reply by maxsharabayko** [reply]
I would say it is not supported by the library. But you can check the `CChannel` class to see if you can add it.

**Reply by code101-marko** [reply]
Thanks, will have a look at it.

**Comment by ravtr** [comment]
Thanks, @maxsharabayko.

Are you aware of any workflows using high bitrates?
Are there any benchmarks wrt bandwidth and latency?
